PhysicsTools/NanoAOD/plugins/NanoAODOutputModule.cc

```diff
@@ -299,41 +327,42 @@ void NanoAODOutputModule::openFile(edm::FileBlock const&) {
     m_file->SetCompressionAlgorithm(ROOT::kZLIB);
   } else if (m_compressionAlgorithm == std::string("LZMA")) {
     m_file->SetCompressionAlgorithm(ROOT::kLZMA);
-  } else if (m_compressionAlgorithm == std::string("ZSTD")) {
```

Comment:
Is this an artifact of some merge history issue or do we want to remove these compression algos?

Reply:
That must be a merge mistake, thanks for spotting!